guidance/_ast.py

```diff
@@ -683,6 +685,11 @@ def visit(self, node: GrammarNode, top=False) -> str:
                     and target.is_allowed_in_lark_terminal
                     and not node.is_allowed_in_lark_terminal
                 ):
+                    # Handle issue #1320, where a large select() was
```

Comment:
Would welcome suggestions for improving this.... my understanding is slightly hazy